core/arch/aarch64/encode_gen.h

```diff
@@ -6445,9 +6499,9 @@ encoder(byte *pc, instr_t *instr)
     case OP_bcond:
         return encode_opnds_bcond(pc, instr, 0x54000000);
     case OP_bfm:
-        if ((enc = encode_opndsgen_13000000(pc, instr, 0x33000000)) != ENCFAIL)
+        if ((enc = encode_opndsgen_33000000(pc, instr, 0x33000000)) != ENCFAIL)
```

Comment:
This file is autogenerated by codec.py. I'll have to update the script and this will create a big diff.